src/all/hitomi/src/eu/kanade/tachiyomi/extension/all/hitomi/Hitomi.kt

```diff
@@ -190,14 +223,23 @@ class Hitomi(
                 filterNegative(it.await())
             }
 
+            if (random) results = results.shuffle()
```

Comment:
I am not sure about this random sort order...

Reply:
Is it not allowed? The website has that in `Order by:`

Reply:
set is already supposed to be unordered

Reply:
Sorry, but what does that mean? I'm not really familiar with code things. Thank you.